posthog/api/event_definition.py

```diff
@@ -119,21 +168,28 @@ def dangerously_get_queryset(self):
 
     def _ordering_params_from_request(
         self,
-    ) -> tuple[str, Literal["ASC", "DESC"]]:
+    ) -> list[tuple[str, Literal["ASC", "DESC"]]]:
```

Comment:
we always made the results here into a list

and then added name when constructing the query

so pulled that into this method instead

(we're not sending multiple params over the API so we don't really need it but felt more clear to support it)